tools/js-sdk-release-tools/src/common/utils.ts

```diff
@@ -298,6 +299,31 @@ export async function resolveOptions(typeSpecDirectory: string): Promise<Exclude
     return options
 }
 
+export function updateApiVersionInTspConfig(typeSpecDirectory: string, apiVersion?: string) {
+    if (!apiVersion || apiVersion === '') {
```

Comment:
apiversion is required, no need to check it
```suggestion
    if (apiVersion === '') {
```